src/dataconnect/types.ts

```diff
@@ -249,3 +249,74 @@ interface ImpersonationUnauthenticated {
 export type Impersonation = ImpersonationAuthenticated | ImpersonationUnauthenticated;
 
 /** End Dataplane Client Types */
+
+/** Start Gemini client types */
+
+export interface CallCloudAiCompanionRequest {
```

Comment:
Let's separate the FDC API's types and Cloud AI Companion (TAF)'s types.

They are quite different~

Reply:
Yeah, would be nice in short term~